supported/osg-htc/osg-hosted-ce/templates/configmap.yaml

```diff
@@ -132,24 +131,6 @@ data:
     JOB_ROUTER_ROUTE_NAMES = Hosted_CE_default_route
     # Operator-provided HTCondor-CE configuration below this line
 {{ .Values.HTCondorCeConfig | default "" | indent 4 }}
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: osg-hosted-ce-{{ .Values.Instance }}-slate-scitokens
-  labels:
-    app: osg-hosted-ce
-    instance: {{ .Values.Instance }}
-    release: {{ .Release.Name }}
-    app.kubernetes.io/part-of: {{ .Chart.Name }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-data:
-  50-slate-scitokens.conf: |+
-    {{- range $index, $map := .Values.SciTokenRemoteUserMapping }}
-    {{- range $url, $user := $map }}
-    SCITOKENS /^{{ $url | replace "/" "\\/" | replace "." "\\." | replace "-" "\\-" }}{{ if not ( $url | contains "," ) }},{{ end }}/ {{ $user }}
-    {{- end }}
-    {{- end }}
```

Comment:
We definitely still need this bit. The name of the ConfigMap can probably be changed (and the ref in the deployment updated) but this is what drives user creation and which organizations are allowed to submit jobs to the CE!

Reply:
Renamed instead of deleted